1주차/14659/14659_java_포키.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int count = scanner.nextInt();
+        scanner.nextLine();
+        List<Integer> peeks = Arrays.stream(scanner.nextLine().split(" "))
+                .map(Integer::parseInt)
+                .collect(Collectors.toList());
```

Comment:
저는 해당 로직을 다음과 같이 풀었어요!! 해당 로직인지 좋은지 확실하지는 못해서 이런 방법도 있었구나 정도로 슥 넘어가시면 될 것 같습니다 😁
```suggestion
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        int N = Integer.parseInt(br.readLine());
        int max_count = 0;

        st = new StringTokenizer(br.readLine());
        List<Integer> peaks_height = new ArrayList<>();
        while (st.hasMoreTokens()) {
            peaks_height.add(Integer.parseInt(st.nextToken()));
        }
```